libs/@guardian/libs/src/performance/startPerformanceMeasure.ts

```diff
@@ -47,16 +52,20 @@ export const startPerformanceMeasure = (
 					duration: performance.now() - start,
 				};
 
-				return Math.ceil(duration);
+				const formattedDuration = Math.ceil(duration);
+
+				logPerf(measureName, formattedDuration);
```

Comment:
“Formatted” or “rounded up to the nearest millisecond”?

Reply:
i'd say "rounded up to the nearest millisecond" is how it's been formatted